bitnami/prometheus-operator/templates/alertmanager/pdb.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if and .Values.alertmanager.enabled .Values.alertmanager.podDisruptionBudget.enabled }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ template "prometheus-operator.alertmanager.fullname" . }}
+  labels: {{- include "prometheus-operator.alertmanager.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app: alertmanager
+      alertmanager: {{ template "prometheus-operator.alertmanager.fullname" . }}
```

Comment:
Are these labels correct?

Reply:
I guess it's the same you previously mentioned: https://github.com/coreos/prometheus-operator/blob/master/pkg/alertmanager/statefulset.go#L195-L197

Reply:
yes. it's the same case as the previous comment